set.mm

```diff
@@ -148416,6 +149653,19 @@ Splicing words (substring replacement)
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
 $)
 
+  $c reverse $.
+
+  $( Syntax for the word reverse operator. $)
```

Comment:
I think a better name for this operation is "word reversal"

Reply:
I did not create this definition, I just moved it to the corresponding subsection. Since this does not belong to the topic of this PR, I do not want to change this (without agreement of the contributor).